product_docs/docs/pgd/5/cli/index.mdx

```diff
@@ -18,27 +18,17 @@ redirects:
 The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool for managing your EDB Postgres Distributed cluster. It's the key tool for inspecting and managing cluster resources.
 
 It allows you to run commands against EDB Postgres Distributed clusters to:
- * Determine the health of the cluster, inspect the cluster's configuration, and manage the cluster's resources. 
- * Inspect and manage the cluster's nodes, groups, and proxies.
- * Perform switchover operations on the write leaders of groups.
+
+* Determine the health of the cluster, inspect the cluster's configuration, and manage the cluster's resources. 
+* Inspect and manage the cluster's nodes, groups, and proxies.
```

Comment:
Probably we don't want to mention `proxies` as we have deprecated the proxy commands

Reply:
got it